google/cloud/opentelemetry/trace_exporter.h

```diff
@@ -27,7 +27,7 @@ namespace cloud {
 namespace otel {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
-// Make an OpenTelemetry Trace Exporter for Google Cloud Trace.
+/// Make an OpenTelemetry Trace Exporter for Google Cloud Trace.
```

Comment:
This is no adequate for a public API, but you need to start somewhere.

Reply:
Yeah. The short description is better than no description.